homatransport/src/transport/VectioTransport.h

```diff
@@ -206,10 +208,40 @@ class VectioTransport : public cSimpleModule
     typedef std::map<uint64_t, std::set<inet::L3Address>> FinishedMsgsMap;
     FinishedMsgsMap finishedMsgs;
 
-    double currentRtt = 2.0 * 5e-6;
-    int allowedInFlightGrantedBytes = ((int)(2.0 * 5e-6 * 10e9 / 8.0));
+    double currentRtt = 2.5 * 2.0 * 1.6e-6;
+
+    // max allowed inflight grant bytes per receiver
+    int allowedInFlightGrantedBytes = ((int)(2.5 * 2.0 * 1.6e-6 * 10e9 / 8.0));
+    int currentRcvInFlightGrantBytes = 0;
+
+    typedef std::map<inet::L3Address,int> SenderInFlightGrantBytes;
+    SenderInFlightGrantBytes senderInFlightGrantBytes;
+
+    // for each sender, stores a pair of current actively granted msg and
+    // bytes remaining to grant
+    typedef std::map<inet::L3Address,std::pair<uint64_t,int>> SenderActiveGrantedMsg;
```

Comment:
prefer "using declaration" to typedef.
using SenderActiveGrantedMsg = std::map<inet::L3Address, std::pair<uint64, int>>;